src/operation/emulation_model_store.jl

```diff
@@ -133,8 +138,8 @@ function write_result!(
     update_timestamp::Dates.DateTime,
     df_row::DataFrames.DataFrameRow,
 )
-    container = getfield(store, get_store_container_type(key))
-    container[key][index, :] = df_row
+    container = getfield(store.data_container, get_store_container_type(key))
+    get_values(container[key])[index, :] = df_row
```

Comment:
Would it be more clear to invert the logic like
```
set_values!(container, key, index, df_row)
```

Reply:
fixed via 927d781e96adf851a8ed98243142c4a4f2e64289